### PR TITLE
Turn out if you use es6 modules use strict is already on!

### PR DESCRIPTION
**Pull request by chrisnojima**
This cleansup a bunch of noise

Also:
Changes one import to make it smaller
Syncs babelrc with the one from react 14.3

### CODE REVIEW

File: react-native/.babelrc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Keep in sync with packager/transformer.js
+// Our overrides
```
Comment (chrisnojima) [inline react-native/.babelrc]: Matches RNs

File: react-native/.eslintrc [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "rules": {
         "standard/object-curly-even-spacing": [2, "never"],
         "standard/array-bracket-even-spacing": [2, "never"],
-        "standard/computed-property-even-spacing": [2, "never"]
+        "standard/computed-property-even-spacing": [2, "never"],
+        "strict": [2, "global"]
```
Comment (chrisnojima) [inline react-native/.eslintrc]: give strict warnings

File: react-native/react/common-adapters/native-event-emitter.desktop.js [hunk 3]
```diff
@@ -1,5 +1,2 @@
-'use strict'
-
 import EventEmitter from 'events'
-let RCTDeviceEventEmitter = new EventEmitter()
-module.exports = RCTDeviceEventEmitter
+export default new EventEmitter()
```
Comment (chrisnojima) [inline react-native/react/common-adapters/native-event-emitter.desktop.js]: let's always use export

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**